fotogalleri/backend/models.py

```diff
@@ -1,3 +1,56 @@
 from django.db import models
+from os import path
 
-# Create your models here.
+
+def new_image_path(instance, filename):
+    '''
+    Image will be uploaded to MEDIA_ROOT/<input_dirs>/<filename>
+    '''
+    return path.join(instance.input_dirs, filename)
+
+
+class ImageMetadata(models.Model):
+    '''
+    Metadata model for uploaded images.
+
+    Stores simple metadata such as file name, path, and time and date of upload.
+    '''
+    filename = models.CharField(max_length=255)
+    image_path = models.ImageField(upload_to=new_image_path)
+    upload_time = models.DateTimeField(auto_now_add=True, blank=True)
```

Comment:
Just a detail, but this field name might be confusing (time it took to upload?). I think "created_at" is better naming for a timestamp.

Reply:
+1